packages/@webex/plugin-cc/src/AgentConfigService/AgentConfigService.ts

```diff
@@ -0,0 +1,140 @@
+      const URL = `${this.wccAPIURL}organization/${this.orgId}/agent-profile/${desktopProfileId}`;
+      const response = await this.requestInstance.request(URL, HTTP_METHODS.GET);
+
+      if (response.statusCode !== 200) {
+        throw new Error(`getDesktopProfileById api failed. Error: ${response}`);
+      }
+
+      this.webex.logger.log('getDesktopProfileById api success.');
+
+      return Promise.resolve(response.body);
+    } catch (error) {
+      return Promise.reject(new Error(`getDesktopProfileById api failed. Error: ${error}`));
+    }
+  }
+
+  /**
+   * Method to get List of Teams.
+   * @param {number} page Index of the page of results to be fetched. Defaults to 0.
+   * @param {number} pageSize Number of items to be displayed on a page. Defaults to 10.
+   * @param {Array<String>} filter Filter that can be applied to the elements to be fetched. Defaults to [].
+   * @param {Array<String>} attributes Specify the attributes to be returned. Defaults to ['id'].
+   * @returns {Promise<ListTeamsResponse>} A promise that eventually resolves to an API response.
+   */
+
+  public async getListOfTeams(
+    page: number,
+    pageSize: number,
+    filter: string[],
+    attributes: string[]
+  ): Promise<ListTeamsResponse> {
+    try {
+      let URL = '';
+      if (filter && filter.length > 0)
+        URL = `${this.wccAPIURL}organization/${this.orgId}/team?page=${page}&pageSize=${pageSize}&filter=id=in=${filter}&attributes=${attributes}`;
+      else
+        URL = `${this.wccAPIURL}organization/${this.orgId}/team?page=${page}&pageSize=${pageSize}&attributes=${attributes}`;
+      const response = await this.requestInstance.request(URL, HTTP_METHODS.GET);
+
+      if (response.statusCode !== 200) {
+        throw new Error(`getListOfTeams api failed. Error: ${response}`);
+      }
+
+      this.webex.logger.log('getListOfTeams api success.');
+
+      return Promise.resolve(response.body);
+    } catch (error) {
+      return Promise.reject(new Error(`getListOfTeams api failed. Error: ${error}`));
+    }
+  }
+
+  /**
+   * Method to get List of AuxCodes.
+   * @param {number} page Index of the page of results to be fetched. Defaults to 0.
+   * @param {number} pageSize Number of items to be displayed on a page. Defaults to 10.
+   * @param {Array<String>} filter Filter that can be applied to the elements to be fetched. Defaults to [].
+   * @param {Array<String>} attributes Specify the attributes to be returned. Defaults to ['id'].
+   * @returns {Promise<ListAuxCodesResponse>} A promise that eventually resolves to an API response.
+   */
+
+  public async getListOfAuxCodes(
+    page: number,
+    pageSize: number,
+    filter: string[],
+    attributes: string[]
+  ): Promise<ListAuxCodesResponse> {
+    try {
+      let URL = '';
+      if (filter && filter.length > 0)
+        URL = `${this.wccAPIURL}organization/${this.orgId}/team?page=${page}&pageSize=${pageSize}&filter=id=in=${filter}&attributes=${attributes}`;
```

Comment:
why are we calling teams here? we need to call aux codes. Did we test? are aux codes coming?

Reply:
Code Updated!